.eslintrc.js

```diff
@@ -12,7 +12,9 @@
   ],
   "rules": {
     "@typescript-eslint/array-type": "error",
-    "@typescript-eslint/explicit-function-return-type": "error",
+    "@typescript-eslint/explicit-function-return-type": ['on', 
+    {allowTypedFunctionExpressions: true
+    }],
```

Comment:
what does this change do?

Reply:
Reverted - suggestion from someone at Zoopla

Reply:
If I'm reading the documentation correctly they actually do the same thing anyway!
I agree that we want `allowTypedFunctionExpressions` to be true... but that's should be the default in any case.